html/rendering/non-replaced-elements/the-fieldset-element-0/fieldset-containing-block-ref.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<title>Reference for fieldset containing block</title>
+<style>
+p { margin: 0; height: 100px }
+div { position: absolute; top: 108; width: 100px; height: 100px; background: lime; }
```

Comment:
Presumably this should be 108px? If this worked regardless, something seems off?

Reply:
It gets positioned where it would be if it was statically positioned. But it should say px of course.

Reply:
fixed